composer.json

```diff
@@ -33,5 +32,11 @@
         "psr-4": {
             "yii\\Psr7\\tests\\": "tests/"
         }
-    }
+    },
```

Comment:
Can you provide an updated `config/web.php` section that allows this to work without yii2-bower-asset?

What is documented in the repository is:

```
 'aliases' => [
        '@bower' => '@vendor/yidas/yii2-bower-asset/bower',
        '@npm'   => '@vendor/npm-asset',
    ],
```

Is that not needed anymore or does this eliminate the need for yii2-bower-asset entirely?

If you can provide a functional updated example in the main MR body I can review it.

Reply:
that is a standart way to work with assets in Yii2, no plugins required https://github.com/yiisoft/yii2-app-basic/blob/master/config/web.php#L10